template/default.tmpl

```diff
@@ -1,5 +1,5 @@
 {{ define "__alertmanager" }}AlertManager{{ end }}
-{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts{{ end }}
+{{ define "__alertmanagerURL" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}{{ end }}
```

Comment:
Or even if the filtering is done in the frontend, the URL param for `receiver=...` should be a regexp in any case, so the above is true in any case.